boost_python/compression.cc

```diff
@@ -117,6 +118,9 @@ unsigned int dxtbx::boost_python::cbf_decompress(const char *packed,
   bool le = little_endian();
 
   while (j < packed_sz) {
+    if ((values - original) > unpacked_sz) {
+      break;
+    }
```

Comment:
should this `return 0`?

Reply:
🤔

TBH I think this should probably just raise an exception here, but I am not sure

It probably _should not_ return 0 as it _has_ unpacked a whole image - what we should do with trailing garbage is an interesting question

Reply:
I think "number read" is fine - in theory you might want to know
- if you got less than expected
- If you filled a buffer if for some reason you don't know how long
Maybe there's a better signal e.g. returning one larger but that sounds like it'd just cause problems when forgetting. This probably isn't used enough for it to be a problem.